platform/security/pdp/security-pdp-xacmlrealm/pom.xml

```diff
@@ -25,6 +27,16 @@
 
     <dependencies>
```

Comment:
I won't have time to properly review this until tomorrow or Monday, but this class shouldn't be included in this PR.